### PR TITLE
Update OpenSSL version for libgit2

### PR DESCRIPTION
**Pull request by uilianries**
Specify library name and version:  **libgit2/0.28.3**

Update OpenSSL version to avoid broken package: https://github.com/conan-io/examples/pull/61#issuecomment-614692581

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by danimtb** [comment]
Blocked by #1386

**Comment by jgsogo** [comment]
Running CI again, it should work now.

File: recipes/libgit2/all/conanfile.py [hunk 1]
```diff
@@ -65,11 +65,11 @@ def _need_mbedtls(self):
...
     def requirements(self):
         self.requires("zlib/1.2.11")
-        self.requires("http_parser/2.9.2")
+        self.requires("http_parser/2.9.4")
         if self.options.with_libssh2:
-            self.requires("libssh2/1.8.2")
+            self.requires("libssh2/1.9.0")
         if self._need_openssl:
-            self.requires("openssl/1.1.1d")
+            self.requires("openssl/1.1.1f")
```
Comment (danimtb) [inline recipes/libgit2/all/conanfile.py:72]: Is openssl/1.1.1g already in place?

Comment (uilianries) [inline recipes/libgit2/all/conanfile.py:72]:
```suggestion
            self.requires("openssl/1.1.1g")
```

**Pull request merged**